bin/export-record-barcodes

```diff
@@ -85,20 +189,46 @@ def fetch_records(project):
             for event in project._fetch("event") }
 
     for record in project.records(fields=FIELDS, raw=True):
+        arm = event_arm[record["redcap_event_name"]]
 
-        query = urlencode({
+        query_params = {
             "pid": project.id,
-            "arm": event_arm[record["redcap_event_name"]],
+            "arm": arm,
             "id": record.id,
-        })
+        }
+
+        # The PCDEQC form is only for swab-and-send kits. Only deep link to the
+        # PCDEQC form if it exists and if the swab was not collected in-person
+        # at a kiosk. All of the SCAN PH and SCAN IRB projects are swab-and-send
+        # except for the SCAN In-person enrollments project which does not have
+        # the PCDEQC form. Therefore, we only need to check the UW Reopening
+        # (i.e. Husky Coronavirus Testing) REDCap project to determine if the
+        # barcode is a kiosk barcode.
+        if 'post_collection_data_entry_qc_complete' in record \
+            and not record.get('collect_barcode_kiosk'):
```

Comment:
PCDEQC does exist in the SCAN In-Person enrollments project, so this logic needs to be updated. Currently when I look up a barcode within the In-Person project, it does not redirect to the record.

Reply:
I'm not sure if its an issue specific to SCAN in person. For samples being unboxed, when the lab looks up the barcode and wants to fill out the PCDEQC form for that record, wouldn't the record have "post_collection_data_entry_qc_complete" == false?

Reply:
@kschwabefry I believe the main use is by unboxers, but the lab may also use the switchboard to troubleshoot records that have already been QC'ed

Reply:
Thats a great point @joverlee521

Reply:
I pushed a new fix! The solution appears to be as simple as this (because `post_collection_data_entry_qc_complete` exists in all projects even if no records have a PCDEQC form initiated). 
* If `post_collection_data_entry_qc_complete` is an empty string, redirect to the record home page 
* otherwise, redirect to the PCDEQC form.

Reply:
Thank you for digging! It links to to the correct record/instance for In-Person, SCAN, and UW-reopening projects!